features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/SessionReplayRecorder.kt

```diff
@@ -104,8 +109,23 @@ internal class SessionReplayRecorder : OnWindowRefreshedCallback, Recorder {
         this.recordedDataQueueHandler = RecordedDataQueueHandler(
             processor = processor,
             rumContextDataHandler = rumContextDataHandler,
-            timeProvider = timeProvider,
-            internalLogger = internalLogger
+            internalLogger = internalLogger,
+
+            /**
+             * TODO RUMM-0000 consider change to LoggingThreadPoolExecutor once V2 is merged.
```

Comment:
> [!CAUTION]
> Do not use a "RUM-000" ticket, actually create one and reference it here.

Reply:
👍  I've opened a ticket and modified this to reference